docker-assets/auth-api-service.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+export PATH=$PATH:/opt/rubies/ruby-2.3.1/bin
```

Comment:
Is this not something that is handled by the `container-environment` file?

It's defined in the dockerfile so I figure it should get dumped into that file.